src/services/MnemonicSave.ts

```diff
@@ -0,0 +1,66 @@
+import { Storage } from "@utils/StorageArray";
+
+const CHECK_NEEDED_KEY = "mnemonic-check-needed";
+
+interface WordChecked {
+  word: string;
+  isDisabled: boolean;
+}
```

Comment:
Exposing this here feels like leaking implementation details of the UI into the service.